src/bytes.rs

```diff
@@ -902,6 +921,17 @@ static SHARED_VTABLE: Vtable = Vtable {
     drop: shared_drop,
 };
 
+struct SharedInline {
+    ref_cnt: AtomicUsize,
+    cap: usize,
+    // data: [u8; cap]
```

Comment:
Can this just be an actual DST? Admittedly I haven't tried to use that part of Rust much...

Reply:
@seanmonstar it can't because `*mut SharedInline` must be one word size (to fit into `AtomicPtr()`), and DST is two words.